### PR TITLE
popper.js: update type definitions to 1.8.5

### PR DESCRIPTION
**Pull request by joscha**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/FezVrasta/popper.js/issues/62, https://github.com/FezVrasta/popper.js/commit/2c20861ea17a7090937654852439e178bc16f14e, https://github.com/FezVrasta/popper.js/issues/239
- [x] Increase the version number in the header if appropriate.

cc @FezVrasta @mattrcarter

### CODE REVIEW

File: types/popper.js/index.d.ts
Comment (FezVrasta) [inline types/popper.js/index.d.ts]: Note that this is deprecated, now it's just `fn`. The old one is still supported tho

Comment (joscha) [inline types/popper.js/index.d.ts]: thanks! Added in 7091e94e7a99e2c10d6c6a1376302c130a70e6fb.

File: types/popper.js/index.d.ts
Comment (rhysd) [inline types/popper.js/index.d.ts]: I think copyright notice should be added after other headers because some bots look `Type definitions for ...` header.

Comment (rhysd) [inline types/popper.js/index.d.ts]: This copyright notice should be removed because all files are served with license at `DefinitelyTyped/LICENSE`.

**Review by rhysd — CHANGES_REQUESTED** [review]

File: types/popper.js/index.d.ts
Comment (joscha) [inline types/popper.js/index.d.ts]: ah, an accident - automatically inserted by the editor, will remove!

Comment (joscha) [inline types/popper.js/index.d.ts]: removed in 49b2040

**Review by rhysd — APPROVED** [review]

File: types/popper.js/index.d.ts
Comment (rhysd) [inline types/popper.js/index.d.ts]: Thanks!

**Comment by joscha** [comment]
Thanks @rhysd! Do you know what the process is after this to get this merged? I can't tick the checkbox in the bot comment if that's what triggers it?

**Comment by rhysd** [comment]
The checkbox is checked by committer. So you and I can't do anything. Let's wait for committer visiting this PR.

**Pull request merged**
